controllers/etcdbackupschedule_controller.go

```diff
@@ -28,13 +29,45 @@ type EtcdBackupScheduleReconciler struct {
 	CronHandler CronScheduler
 
 	// Schedules holds a mapping of resources to the object responsible for scheduling the backup to be taken.
-	Schedules map[string]Schedule
+	Schedules *ScheduleMap
 }
 
 type Schedule struct {
 	cronEntry cron.EntryID
 }
 
+// ScheduleMap is a thread-safe mapping of backup schedules.
+type ScheduleMap struct {
+	sync.RWMutex
+	data map[string]Schedule
+}
+
+func NewScheduleMap() *ScheduleMap {
+	return &ScheduleMap{
+		RWMutex: sync.RWMutex{},
```

Comment:
nit: It might be possible to omit this, as the zero-value is an unlocked mutex: https://golang.org/pkg/sync/#RWMutex